.github/scripts/yaml-tests/failing-runtime-files/hwdb-invalid-afi/sample_config_hwdb.yaml

```diff
@@ -1,4 +0,0 @@
-testing-hwdb-entry:
-    agfi: INVALID_AGFI
```

Comment:
@abejgonzalez , This test also seems like something that shouldn't be caught by firesim before asking AWS to really do something with the AGFI.  I'm going to leave it out